pkg/types/v1/config.go

```diff
@@ -541,8 +541,11 @@ func (i *UpgradeUkiSpec) ShouldReboot() bool   { return i.Reboot }
 func (i *UpgradeUkiSpec) ShouldShutdown() bool { return i.PowerOff }
 
 type ResetUkiSpec struct {
-	Reboot   bool `yaml:"reboot,omitempty" mapstructure:"reboot"`
-	PowerOff bool `yaml:"poweroff,omitempty" mapstructure:"poweroff"`
+	FormatPersistent bool `yaml:"reset-persistent,omitempty" mapstructure:"reset-persistent"`
```

Comment:
again probably lack of knowledge on the whole installation so bear with me ... same applies for any other comment here so I don't repeat myself 🐼 

why do we use different naming here? it's the first time I read about Format, reset on the other hand is quite common to find.

Reply:
This is unfortunately coming from the normal reset where we called it reset-oem/reset-persistent and needs to be kept for the configs and such.
But the real action that we are doing is formatting the partition. reset means nothing for persistent/oem as we just format the partition, so I though it may be clearer to have it like that so it also syncvs with the normal reset in which we have the same values.

I mean, `if FormatPersistent { FormatPartition(spec.Partitions.Persistent) }` seems nicer than `if ResetPersistent { FormatPartition(spec.Partitions.Persistent) }` as the second TO ME looks like more things are gonna be done to that persistent thing to "reset" it :rofl: 

But it could be me, I dont mind changing it.

Reply:
I like your approach, makes total sense if this is the actual action that gets executed, it's unfortunate it will be confusing but I don't think it's worth carrying the bad name over. And I'd say we should try to keep track of these things to long-term switch `reset-persisten` to `format-persistent` and deprecate the old names (same for any other case like this)